contract/test/utilities/deployAll.ts

```diff
@@ -0,0 +1,93 @@
+// import { HardhatRuntimeEnvironment } from 'hardhat/types'
+import hre, { ethers } from 'hardhat'
+
+export async function deploy (context) {
+  // Deploy Price Oracles
+  const ORACLE_UNIT_PRICE = parseInt(process.env.ORACLE_PRICE_PER_SECOND_IN_WEIS || '3')
```

Comment:
I updated oracles. You might need to update this part

Reply:
and similar code that references oracles

Reply:
Done.
Have merged your upstream changes.